app/templates/partials/summary/answer-summary.html

```diff
@@ -0,0 +1,33 @@
+<div class="summary">
+  <h2 class="summary__title neptune">{{ content.summary.label }}</h2>
+  {%- for group in content.summary.groups -%}
+
+  <div class="summary__block u-mb-l">
+    {%- for answer in group.answers -%}
+    <div class="summary__item">
+      <div class="grid">
+        <div class="grid__col col-8@xs mars {{ 'col-10@m' }}">
+          <div class="summary__label mars" id="{{ answer.id }}-{{ loop.index }}-label" data-qa="{{ answer.id }}-{{ loop.index }}-label">
+            <p class="icon--{{content.summary.icon}}" data-qa="{{ answer.id }}-{{ loop.index }}">
+              {{ answer.label|striptags }}
+            </p>
+          </div>
+        </div>
+        <div class="grid__col col-4@xs col-2@m">
+          <div class="summary__link mars">
+            <a href="{{ answer.block.link }}#{{ answer.id }}" class="summary__edit-link"
+               aria-describedby="{{ answer.id }} {{ answer.id }}-answer" data-qa="{{ answer.id }}-{{ loop.index }}-edit" {{
+               helpers.track('click', 'Summary', 'Edit click') }}>{{ _("Change") }}
+            <span class="u-vh">
+                            {{ _("Change your answer for") }}: {{ answer.label|striptags }}. {{ _( "The current value is") }}:
+                            {{ answer.label|striptags }}
```

Comment:
Spoke with @ajmaddaford and he confirmed that we should use this style for translation:
https://github.com/ONSdigital/eq-survey-runner/blob/1cc1406becda068f3519bd34d36ee4bd7cbbb800/app/templates/partials/summary/edit-link.html#L7-L9